Code/GraphMol/MolHash/hashfunctions.cpp

```diff
@@ -509,12 +509,23 @@ std::string TautomerHashv2(RWMol *mol, bool proto, bool useCXSmiles,
                   << atm->getIdx() << "-" << oatom->getIdx() << std::endl;
         std::cerr << "    " << bondsConsidered[nbrBond->getIdx()] << " icao "
                   << isCandidateAtom(oatom) << " hsbo "
-                  << hasStartBond(oatom, startBonds) << " unsat "
+                  << hasStartBond(oatom, startBonds) << " atomunsato "
+                  << queryAtomUnsaturated(oatom) << " atomunsat "
+                  << queryAtomUnsaturated(atm) << " bondunsat "
```

Comment:
atomunsato atomunsat atomunsa unsatoatom unsatatom unsaatom unatomsa ...

Reply:
I really went all out on the creativity for those debug messages, eh?